definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -3083,5 +3083,17 @@
     "PostConditionState": "*",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/return-to-state-before-general-referral",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "21/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "Move_To_ServiceAppRejected",
+    "Name": "Move to service app rejected",
+    "Description": "Move to service app rejected state",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "AosAwaiting",
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
this can be removed

Reply:
Do you mean the "RetriesTimeoutURLAboutToSubmitEvent"?

Reply:
yes, this is completely ignored by CCD. Used only for callbacks. There is no callback here. And it's ignored anyway :)